### PR TITLE
fix(ndt_scan_matcher): fix default parameter to 0.0225

### PR DESCRIPTION
**Pull request by kminoda**
Signed-off-by: kminoda <koji.m.minoda@gmail.com>

## Description
Change default parameter from 0.025 to 0.0225 (= 0.15^2).
It is misleading to set 0.025 as a default covariance as its square root is an irrational number. Instead, I propose to set it to 0.0225 so that its sqrt will also be a rational number too.

If 0.025 was set based on some experiments and shall not be changed without any quantitative evaluation, I will close this PR.

Related: https://github.com/tier4/autoware_launch/pull/438

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Review by YamatoAndo — APPROVED** [review]

**Comment by yukkysaito** [comment]
>If 0.025 was set based on some experiments

@kminoda 
It's ok. But can you add the comment to the yaml?

**Comment by kminoda** [comment]
@yukkysaito What kind of comment do you have in your mind?

**Comment by yukkysaito** [comment]
@kminoda this is example :+1: 
```
# 0.025 was set based on some experiments without any quantitative evaluation,
```

**Comment by kminoda** [comment]
@yukkysaito I got the point, thanks! I added a comment as follows
```
# Do note that this covariance matrix is empirically derived
```

File: launch/tier4_localization_launch/config/ndt_scan_matcher.param.yaml [hunk 1]
```diff
@@ -55,9 +55,9 @@
     # The covariance of output pose
```
Comment (yukkysaito) [inline launch/tier4_localization_launch/config/ndt_scan_matcher.param.yaml:55]:
```suggestion
    # The covariance of output pose
    # Do note that this covariance matrix is empirically derived
```

Comment (kminoda) [reply launch/tier4_localization_launch/config/ndt_scan_matcher.param.yaml:55]: Right, thanks.
https://github.com/autowarefoundation/autoware.universe/pull/1583/commits/b9d8976b2cfa3b6af5a51c4eff1851b294680f76

**Review by yukkysaito — APPROVED** [review]
LGTM

**Pull request merged**